circleguard/settings.py

```diff
@@ -322,6 +328,10 @@ def get_setting(name):
     # second bullet here: https://doc.qt.io/qt-5/qsettings.html#platform-limitations
     if type_ is bool:
         return False if val in ["false", "False"] else bool(val)
+    # if type_ is list:
```

Comment:
Why is this a comment?

Reply:
thought I removed this, I was using json.loads because I would list wasn't inherently supported by qt but turns out it is so we don't need it

Reply:
although actually this (uncommented) code might be necessary for windows since the registry doesn't preserve type

Reply:
Seems to be stored correctly as uncommenting it breaks it
![](https://i.imgur.com/ps2WvIw.png)
![](https://i.imgur.com/yFEkHvH.png)